src/App.js

```diff
@@ -4,20 +4,37 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  handleInputChange = (inputCganged) => {
```

Comment:
это **всегда** `event`
```suggestion
  handleInputChange = (event) => {
```